tests/test_scheduler_query.py

```
@@ -0,0 +1,113 @@
+    )
+    session.add(match)
+    await session.commit()
+    await session.refresh(match)
+
+    # Add a result for this match
+    result = Result(match_id=match.id, winner="Team C", score="2-0")
+    session.add(result)
+    await session.commit()
+
+    # Execute the function under test
+    fetched_now, matches = await _get_matches_starting_soon(session)
```

Ruff (0.14.10)

Unpacked variable `fetched_now` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)